### ISSUE TITLE
Examples on http://www.glvisualize.com/ don't work

### ISSUE THREAD

**Issue by MRedies** [JuliaGL/GLVisualize.jl#178]
It seems most of the examples on http://www.glvisualize.com/ don't work. Most of the time the issue seems to be replacing 

`view(barsvis, window)`
with

`_view(barsvis, window)`

but finding this is very cumbersome for beginners (like me).

**Comment by SimonDanisch** [comment]
I'm really sorry about this, this has been lingering for way too long...
Please stick to the examples folder in GLVisualize directly. It should contain all the examples from glvisualize.com, and even some more! ;)

**Comment by tshort** [comment]
Also, glvisualize.com seems down now.

**Comment by hustf** [comment]
Julia is going to stabilize over the next few years, and five years of
effort is going to pay off. Good luck and dont be sorry!


Den tir. 6. mar. 2018, 21:11 skrev Tom Short <notifications@github.com>:

**Comment by SimonDanisch** [comment]
Yeah, I'm currently concentrating on making makie work well, sorry about these issues! One should rather look into the example folder and go through Pkg.runtest("GLVisualize") anyways.
